lib/data/translators.yml

```diff
@@ -53,6 +52,7 @@ namibia: /government/publications/namibia-list-of-lawyers
 nepal: /government/publications/nepal-list-of-lawyers
 netherlands: /government/publications/netherlands-list-of-lawyers
 nicaragua: /government/publications/nicaragua-list-of-lawyers
+north-macedonia: /government/publications/macedonia-list-of-lawyers
```

Comment:
does this also need updating? it redirects to
`https://www.gov.uk/government/publications/north-macedonia-lawyers`

Reply:
Yup, same as above. 😄